input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,295 @@ Example: Patient resource with interpreter required and language is known
 }
 ~~~
 
+### Representing body site and laterality
+When using some AU Core profiles it may be desirable to represent relevant body site and laterality information associated with a record using CodeableConcept elements, notably:
+*  [AU Core Condition](StructureDefinition-au-core-condition.html) - with primary finding code `Condition.code` and `Condition.bodySite`
+*  [AU Core Procedure](StructureDefinition-au-core-procedure.html) - with primary procedure code `Procedure.code` and `Procedure.bodySite`
+
+When recommending approaches to this, considerations include:
+* The `bodySite` element is NOT `Must Support` in AU Core profiles, there is no expectation to fill or meaningfully consume this element.
+* The `CodeableConcept.text` element is system populated and may reflect more specific detail than the `CodeableConcept.coding` concepts provided.
+
+Body site and associated laterality may be recorded in various ways. Depending on the specific system approach to body site / laterality related content this can lead to a number of scenarios needing to be supported with using AU Core profiles:
```

Comment:
Wording suggestion: Replace "Body site and associated laterality may" with "In FHIR, body site and associated laterality can"